colossalai/shardformer/policies/llama.py

```diff
@@ -385,6 +414,10 @@ def module_policy(self):
         from transformers import LlamaForSequenceClassification
 
         policy = super().module_policy()
+        if self.pipeline_stage_manager:
+            use_zbv = self.pipeline_stage_manager.use_zbv
+        else:
+            use_zbv = False
```

Comment:
`use_zbv = self.pipeline_stage_manager is not None and self.pipeline_stage_manager.use_zbv`